src/main/ruby/core/numeric_mirror.rb

```diff
@@ -47,15 +47,17 @@ def step_float_size(value, limit, step, asc)
         end
       end
 
-      def step_size(limit, step)
-        values = step_fetch_args(limit, step)
+      def step_size(limit, step, by)
```

Comment:
Mmh, this sounds like a bug in MRI.
I found something in the documentation of Numeric#step though:
```
In the fixed position argument style, zero
as a step (i.e. num.step(limit, 0)) is not allowed for historical
compatibility reasons.
```